src/terrain/routes.clj

```diff
@@ -167,6 +167,7 @@
 (defn unsecured-routes
   []
   (util/flagged-routes
+   (admin-token-routes)
```

Comment:
I considered putting `admin-token-routes` under a separate group of routes with a different context, but ultimately decided against it. This works, and I don't think it's overly confusing.